client/src/components/existing-request/existing-request-search.vue

```diff
@@ -145,21 +149,17 @@ export default class ExistingRequestSearch extends Vue {
 }
 </script>
 
-<style lang="sass" scoped>
-.col
-  padding: 0
-  margin-top: 1.5rem
-
-.forgot-nr-link
```

Comment:
Got rid of obsolete styles. And changed to SCSS!

Reply:
nice!